packages/frontend/web/pages/Article.tsx

```diff
@@ -46,49 +48,53 @@ const overflowHidden = css`
 
 export const Article: React.FC<{
     data: ArticleProps;
-}> = ({ data }) => (
-    <div className={overflowHidden}>
-        <Header
-            nav={data.NAV}
-            pillar={data.CAPI.pillar}
-            edition={data.CAPI.editionId}
-        />
-        <main>
-            <Container borders={true} className={articleContainer}>
-                <article>
-                    <ArticleBody CAPI={data.CAPI} config={data.config} />
-                    <div className={secondaryColumn} />
-                </article>
-            </Container>
-            <OutbrainContainer config={data.config} />
-            <Container
-                borders={true}
-                className={cx(
-                    articleContainer,
-                    css`
-                        border-top: 1px solid ${palette.neutral[86]};
-                    `,
-                )}
-            >
-                <MostViewed sectionName={data.CAPI.sectionName} />
-            </Container>
-        </main>
-
-        <SubNav
-            subnav={data.NAV.subNavSections}
-            pillar={data.CAPI.pillar}
-            currentNavLink={data.NAV.currentNavLink}
-        />
-        <BackToTop />
-
-        <Footer
-            nav={data.NAV}
-            edition={data.CAPI.editionId}
-            pageFooter={data.CAPI.pageFooter}
-            pillar={data.CAPI.pillar}
-            pillars={data.NAV.pillars}
-        />
-
-        <CookieBanner />
-    </div>
-);
+}> = ({ data }) => {
+    return (
+        <div className={overflowHidden}>
+            <Header
+                nav={data.NAV}
+                pillar={data.CAPI.pillar}
+                edition={data.CAPI.editionId}
+            />
+            <main>
+                <Container borders={true} className={articleContainer}>
+                    <article>
+                        <ArticleBody CAPI={data.CAPI} config={data.config} />
+                        <div className={secondaryColumn}>
+                            <AdSlot
+                                asps={adSlotParameters()}
```

Comment:
We are passing default parameters for the moment. This is a refactoring of the existing prototype.